src/core_modules/capture-core/components/WidgetTwoEventWorkspace/OverflowMenu/Modal/UnlinkModal.js

```diff
@@ -0,0 +1,71 @@
+// @flow
+import React, { useState } from 'react';
+import i18n from '@dhis2/d2-i18n';
+import {
+    Modal,
+    ModalContent,
+    ModalTitle,
+    ModalActions,
+    ButtonStrip,
+    Button,
+    NoticeBox,
+} from '@dhis2/ui';
+import { useDeleteRelationship } from './useDeleteRelationship';
+import type { Props } from './UnlinkModal.types';
+
+export const UnlinkModal = ({
+    setOpenModal,
+    relationshipId,
+    setUpdateData,
+}: Props) => {
+    const [loading, setLoading] = useState(false);
+    const [errorMessage, setErrorMessage] = useState(null);
+
+    const { onDeleteRelationship } = useDeleteRelationship({ sourceId: relationshipId });
+
+    const handleUnlink = async () => {
+        setLoading(true);
+        setErrorMessage(null);
+        setUpdateData(true);
+
+        try {
+            await onDeleteRelationship({ relationshipId });
+            setOpenModal(false);
+        } catch (error) {
+            setErrorMessage(i18n.t('An error occurred while unlinking the relationship.'));
+            setLoading(false);
+        } finally {
+            setLoading(false);
+            setUpdateData(false);
+        }
+    };
+
+    return (
+        <Modal dataTest="event-unlink-modal">
+            <ModalTitle>
+                {i18n.t('Unlink relationship')}
+            </ModalTitle>
+            <ModalContent>
+                <p>{i18n.t('Are you sure you want to unlink this relationship?')}</p>
```

Comment:
Could we say "Are you sure you want to delete the relationship (or link?) between these two events?" I might need to remind myself of the wording in the modal otherwise, I know we want to avoid DHIS2 terminology if we can but I think we have succumbed to needing to use "events" in this functionality? :D 

And maybe we want to add a note about "This will only delete the relationship, not the other event"?